modules/gathering/src/main/Quote.scala

```diff
@@ -1626,6 +1626,36 @@ object Quote:
       "The good thing about chess is that everyone can learn how to play at any age.",
       "Mariya Muzychuk"
     ),
+    Quote(
+      "It’s not about winning or losing, but of course at the end of the day it's about winning or losing.",
+      "Garry Kasparov"
+    ),
+    Quote(
+      "Ok we can change you know? It's an equal change, I change one attacking piece for one defending piece but I still have enough pieces to mate him.",
+      "Garry Kasparov"
+    ),
+    Quote("King safety is the number one priority.", "Garry Kasparov"),
+    Quote("Magnus is a lethal combination of Fischer and Karpov...", "Garry Kasparov"),
+    Quote(
+      "Then it was not considered much of a surprise that I could beat 32 computers at the same time. To me, that was the golden age. Machines were weak, and my hair was strong.",
+      "Garry Kasparov"
+    ),
+    Quote("Ask the knight on g1 what he thinks about the move f3.", "Edward Gufeld"),
+    Quote("Every chessplayer should have a hobby.", "Savielly Tartakower"),
+    Quote(
+      "Tactics is knowing what to do when there is something to do. Strategy is knowing what to do when there is nothing to do.",
+      "Savielly Tartakower"
```

Comment:
we already have this one

    Quote(
      "The tactician must know what to do whenever something needs doing; the strategist must know what to do when nothing needs doing.",
      "Savielly Tartakower"
    ),